db/models/apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment
```

Comment:
Oh, looking below I see this is a 'dead' file since you have another model elsewhere. You should be careful and not leave old dead code lying around! It can get very confusing as our projects get bigger!